plugins/examples/dynamic-clusters/src/index.tsx

```diff
@@ -1,24 +1,102 @@
-import { Headlamp, registerAppBarAction } from '@kinvolk/headlamp-plugin/lib';
-import { Button } from '@mui/material';
+import { ApiProxy, Headlamp, registerAppBarAction } from '@kinvolk/headlamp-plugin/lib';
+import { Button, Modal, Paper, Stack, TextField, Typography } from '@mui/material';
+import { useEffect, useState } from 'react';
+
```

Comment:
Can you please update the README to explain what it demonstrates. Also maybe some more commentary could be helpful in here to describe what it demonstrates.